code/modules/admin/topic.dm

```diff
@@ -22,6 +22,37 @@
 		message_admins("[usr.key] has attempted to override the admin panel!")
 		return
 
+	if(href_list["openticket"])
```

Comment:
This should be a datum topic

see: https://github.com/discordia-space/CEV-Eris/blob/master/code/datums/topic/admin.dm